test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -60,4 +65,13 @@ public static Person generateTestPerson() {
             return null;
         }
     }
+
+    /**
+     * Asserts whether the text in the two given files are the same.
+     */
+    public static void assertTextFilesEqual(Path path1, Path path2) throws IOException {
+        List<String> list1 = Files.readAllLines(path1, Charset.defaultCharset());
+        List<String> list2 = Files.readAllLines(path2, Charset.defaultCharset());
+        assertEquals(list1, list2);
```

Comment:
> org.junit.ComparisonFailure: expected:<...rivate="false">johnd[@gmail.com</email>
>         <address isPrivate="false">John street, block 123, #01-01</address>
>     </persons>
>     <persons>
>         <name>Betsy Crowe</name>
>         <phone isPrivate="true">1234567</phone>
>         <email isPrivate="false">betsycrowe@gmail.com</email>
>         <address isPrivate="true">Newgate Prison</address>
>         <tagged>friend</tagged>
>         <tagged>crim]inal</tagged>
>     </...> but was:<...rivate="false">johnd[f@gmail.com</email>
>         <address isPrivate="false">John street, block 123, #01-01</address>
>     </persons>
>     <persons>
>         <name>Betsy Crowe</name>
>         <phone isPrivate="true">123456227</phone>
>         <email isPrivate="false">betsycrowe@gmail.com</email>
>         <address isPrivate="true">Newgate Prison</address>
>         <tagged>friend</tagged>
>         <tagged>crimf]inal</tagged>
>     </...>

This is what JUnit gives, the first `Person` I only edited `Email `to wrong data but second one I edited both `Phone `and `Tag `to be wrong data. It is clear that first `Person` only gives one extra line but second gives a lot more. 

Since if it is general practice to compare string and only show differences, I will change to cater to general case.